participant/batch/refreshS3Urls.ts

```diff
@@ -0,0 +1,106 @@
+// A batch script to update s3 presigned urls saved to the database.
+// Intended to be run in a lambda.
+import {
+  listExpiringDocuments,
+  updateDocumentS3Url,
+} from "app/utils/db.server";
+import { getURLFromS3 } from "app/utils/s3.server";
+import logger from "app/utils/logging.server";
+
+// Any interactions with Prisma will be async
+// eslint-disable-next-line @typescript-eslint/require-await
+export async function main() {
+  logger.warn(
+    {
+      location: "refreshS3Urls",
+      type: "main.begin",
+    },
+    "Beginning s3 document upload url refresh"
+  );
+
+  const documentsToRefresh = await listExpiringDocuments();
+  if (documentsToRefresh !== undefined && documentsToRefresh.length > 0) {
+    logger.warn(
+      {
+        location: "refreshS3Urls",
```

Comment:
Maybe `tasks.refreshS3Urls` for these? 
(not putting any feet down, just thinking aloud)

Reply:
I'll do `batch.refreshS3Urls`